FluentTerminal.App/Services/ClipboardService.cs

```diff
@@ -13,7 +13,8 @@ public Task<string> GetText()
             {
                 return content.GetTextAsync().AsTask();
             }
-            return null;
+            // Otherwise return a new task that just sends an empty string.
+            return new Task<string>(new Func<string>(() => ""));
```

Comment:
[Returning Task.FromResult(string.Empty) might be easier](https://msdn.microsoft.com/en-us/library/windows/apps/hh194922(v=vs.105).aspx)

Reply:
Quite possibly. This was yoinked from StackOverflow because modern .NET isn't my forte. 😸 

I'll change it.